source/publishing_data/04_configuring_a_source/connectors/openagenda.rst

```diff
@@ -0,0 +1,58 @@
+OpenAgenda connector
+====================
+
+.. admonition:: Important
+   :class: important
+
+   This connector is not available by default. Please contact Opendatasoft for the activation of this connector on a given Opendatasoft domain.
+
+The OpenAgenda connector extracts events from an agenda, using the version 2 of the OpenAgenda REST API.
+
+
+Creation
+~~~~~~~~
+
+1. Enter a space in the Enter a URL area, when creating a new dataset.
+2. Press Enter, or click the + button.
+3. Select OpenAgenda V2 Extractor from the Type list in the Source configuration area.
+
+Configuration
+~~~~~~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+
+   * * Name
+     * Description
+     * Usage
+   * * Agenda
+     * Unique identifier for the agenda.
+     * This field is required.
+   * * API key
+     * Unique identifier for your OpenAgenda account. You can find your API key in your OpenAgenda account settings.
+     * Providing an API key is optional, but it lets you access OpenAgenda data faster.
```

Comment:
I'm not sure to understand the meaning of "access OpenAgenda faster". Could you be more specific? Were did you find this?

Reply:
I found this piece of info in the story [14901](https://app.clubhouse.io/opendatasoft/story/14901/connecteur-openagenda-v2): "N.B.: les appels se font sans clé d'API également, mais l'API retournera les résultats plus lentement."
If this is not accurate, I can remove this piece of info.

Reply:
Ok, this is accurate, it's indeed written in OpenAgenda documentation. You can leave it as it is then.